src/utils/isSemanticVersion.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+export type SemanticVersion = `${ number }.${ number }.${ number }`;
+
+/**
+ * Checks if the given string is a semantic version.
```

Comment:
A bare "semantic version" sounded a little strange to me.
```suggestion
 * Checks if the given string is a semantic version number.
```